conbench/runner.py

```diff
@@ -246,7 +246,8 @@ def record(self, result, name, error=None, publish=True, **kwargs):
             output,
         ) = self._init(kwargs)
 
-        tags["name"] = name
+        if (tags.get("name") is None) or (tags["name"] == ""):
+            tags["name"] = name
```

Comment:
So we want to error here? Or is this something that is used elsewhere (passing the name and not putting it in the tags?) Or maybe error IFF there's a name in tags and a name in kwargs?

Reply:
The kwarg is used by labs/benchmarks. If we error when both are populated, we'll break the case that spurred this issue. We could error if they're both populated and differ.

Reply:
> We could error if they're both populated and differ.

_nods_ yeah that might be best

Reply:
updated